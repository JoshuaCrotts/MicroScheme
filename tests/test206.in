;
; testing imag-part and real-part
;

(real-part 5+4i)
(imag-part 5+4i)
(real-part (+ 5+4i 6-7.5i))
(imag-part (+ 5+4i 6-7.5i))
