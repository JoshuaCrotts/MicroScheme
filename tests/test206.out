5
4
11
-3.5
